### DISCUSSION TITLE
Limit merges to squashed?

### DISCUSSION THREAD

**Discussion by sean-parent** [hylo-lang/hylo#992] [Q&A]
Per CONTRIBUTING.md, we only want squashed merges. Should we update the settings to only allow squashed merges? 
<img width="818" alt="image" src="https://github.com/hylo-lang/hylo/assets/2279724/831afc92-3720-4660-b939-af3cc7b42c42">

**Comment by dabrahams** [comment]
We decided a while back that we simply wanted to discourage “external” contributors from merging long series of semi-meaningless commits like [this one](https://github.com/hylo-lang/hylo/pull/521/commits), but that we wanted to reserve the right to curate a well-organized change-set (especially one where all commits pass tests) and merge it, so we didn't want to prevent merges.  Everything is up for discussion, though.

**Comment by WalterSmuts** [comment]
> especially one where all commits pass tests

Is this something we want to enforce? I believe we can tell CI to run the tests on each commit. Currently we only test the final commit of a merge.
